### PR TITLE
Enable E2E test running on Travis CI

### PR DESCRIPTION
**Pull request by tigrannajaryan**
- We want Travis build to fail if E2E tests fail.
- Added colorizing of test output for easier diagnostics.
- Renamed "perf-test" makefile target to "e2e-test" (since these
  are not just perf tests but also other types of end-to-end tests).
- Fix TestGeneratorAndBackend to make sure it waits properly for
  the generator to produce results.

Issue: https://github.com/open-telemetry/opentelemetry-service/issues/68

### CODE REVIEW

File: testbed/testbed/load_generator.go [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019, OpenCensus Authors
```
Comment (songy23) [inline testbed/testbed/load_generator.go]:
```suggestion
// Copyright 2019, OpenTelemetry Authors
```

File: testbed/testbed/mock_backend.go [hunk 2]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019, OpenCensus Authors
```
Comment (songy23) [inline testbed/testbed/mock_backend.go]:
```suggestion
// Copyright 2019, OpenTelemetry Authors
```

File: testbed/testbed/mock_backend_test.go [hunk 3]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019, OpenCensus Authors
```
Comment (songy23) [inline testbed/testbed/mock_backend_test.go]:
```suggestion
// Copyright 2019, OpenTelemetry Authors
```

File: testbed/testbed/results.go [hunk 4]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019, OpenCensus Authors
```
Comment (songy23) [inline testbed/testbed/results.go]:
```suggestion
// Copyright 2019, OpenTelemetry Authors
```

File: Makefile [hunk 5]
```diff
@@ -35,16 +35,18 @@ all-srcs:
...
+e2e-test: otelsvc
 	$(MAKE) -C testbed runtests
 
 .PHONY: test
 test:
 	$(GOTEST) $(GOTEST_OPT) $(ALL_PKGS)
 
 .PHONY: travis-ci
-travis-ci: fmt vet lint test-with-cover
+travis-ci: fmt vet lint test-with-cover otelsvc
```
Comment (songy23) [inline Makefile]: Do we want to run E2E test on every commit? Maybe only run it once per day?

Comment (tigrannajaryan) [reply Makefile]: We only have a small number of E2E tests and they are quick to run. I think we should run them on every build for now. We can decide to run them less frequently when they start taking a lot of time.

**Comment by tigrannajaryan** [comment]
image

**Comment by tigrannajaryan** [comment]
image

**Comment by tigrannajaryan** [comment]
Reviewers, please check.

**Pull request merged**